lib/Conversion/TritonGPUToLLVM/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ add_triton_library(TritonGPUToLLVM
     SPMDOpToLLVM.cpp
     DecomposeUnsupportedConversions.cpp
     PrintOpToLLVM.cpp
+    RecordOpToLLVM.cpp
```

Comment:
Ditto. I think it should exist under https://github.com/triton-lang/triton/tree/main/third_party/proton/dialect

Reply:
Updated